File: app/src/main/java/can/siempredelao/f1kotlin/backend/Race.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Race {

    public  String                 season;
    public  String                 round;
    public  String                 url;
    public  String                 raceName;
    @SerializedName("Circuit")
    public  Circuit                circuit;
    public  String                 date;
    public  String                 time;
    @SerializedName("Results")
    private List<Result>           results;
    @SerializedName("QualifyingResults")
    public  List<QualifyingResult> qualifyingResults;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/Race.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class Race(val season: String,
           val round: String,
           val url: String,
           val raceName: String,
           @SerializedName("Circuit") val circuit: Circuit,
           val date: String,
           val time: String,
           @SerializedName("Results") val results: List<Result>,
           @SerializedName("QualifyingResults") val qualifyingResults: List<QualifyingResult>)
